src/main/steps/noResponseJourneySequence.ts

```diff
@@ -44,6 +55,64 @@ export const noResponseJourneySequence: Step[] = [
       }
     },
   },
+  {
+    url: NEW_POSTAL_AND_EMAIL,
+    getNextStep: (data: Partial<CaseWithId>): PageLink => {
+      switch (data.applicant1NoResponseNewEmailAndPostalAddress) {
+        case NoResponseNewEmailOrPostalAddress.NEW_POSTAL:
+        case NoResponseNewEmailOrPostalAddress.BOTH_EMAIL_AND_POSTAL: {
+          return NEW_POSTAL_ADDRESS;
+        }
+        case NoResponseNewEmailOrPostalAddress.NEW_EMAIL: {
+          return NEW_EMAIL;
+        }
+        default: {
+          return HUB_PAGE;
+        }
+      }
+    },
```

Comment:
Is there a scenario we're expecting them to go to the hub page for here? If not we could simplify it slightly:
```suggestion
    getNextStep: (data: Partial<CaseWithId>): PageLink => [
      NoResponseNewEmailOrPostalAddress.NEW_POSTAL:
      NoResponseNewEmailOrPostalAddress.BOTH_EMAIL_AND_POSTAL
    ].includes(data.applicant1NoResponseNewEmailAndPostalAddress)
      ? NEW_POSTAL_ADDRESS;
      : NEW_EMAIL;
    },
```